examples/protocol_specification_ex/http.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: http
+author: fetchai
+version: 0.1.0
+license: Apache-2.0
+description: 'A protocol for HTTP requests and responses.'
+speech_acts:
+  request:
+    method: pt:str
+    url: pt:str
+    version: pt:str
+    headers: pt:str
+    bodyy: pt:bytes
```

Comment:
Due to bugs and design errors in the generator (https://github.com/fetchai/agents-aea/issues/905) we cannot chose `body` as a content and we cannot make it optional. Once this is fixed, we can change to `body: pt:optional[pt:bytes]`.

Reply:
Note also the typo `bodyy` instead of `body`.